packages/twenty-front/src/modules/error-handler/components/AppErrorBoundary.tsx

```diff
@@ -10,8 +9,10 @@ export const AppErrorBoundary = ({ children }: { children: ReactNode }) => {
     // TODO: log error to Sentry
   };
 
+  const theme = useTheme();
+
   return (
-    <ThemeProvider theme={THEME_LIGHT}>
+    <ThemeProvider theme={theme}>
```

Comment:
@thaisguigon is this ThemeProvider needed at all here? It seems to be working without but I may be missing something. It was only added recently https://github.com/twentyhq/twenty/pull/4820/files#diff-ad491d07c40dfa8edb12bd537efcd4263bc822f77c4883aebe2b71224e1989cc